### ISSUE TITLE
Terminate microphone access after uploading recording

### ISSUE THREAD

**Issue by flozender** [adamhannadev/quartet#72]

**Comment by adamhannadev** [comment]
I don't think we should worry about this since many users will probably do more than one recording in a session.

**Comment by flozender** [comment]
@adamhannadev can we not stop the microphone access one the recording component unmounts? Because even when you're on the dashboard, the mic is being accessed, that might be concerning for most users.

**Comment by adamhannadev** [comment]
I think I've found a solution. It's not instant though.
